video/bink_decoder.cpp

```diff
@@ -287,39 +287,41 @@ BinkDecoder::BinkVideoTrack::BinkVideoTrack(uint32 width, uint32 height, const G
 	_surface.w = width;
 
 	// Compute the video dimensions in blocks
-	_yBlockWidth   = (width  +  7) >> 3;
-	_yBlockHeight  = (height +  7) >> 3;
-	_uvBlockWidth  = (width  + 15) >> 4;
+	_yBlockWidth = (width + 7) >> 3;
+	_yBlockHeight = (height + 7) >> 3;
+	_uvBlockWidth = (width + 15) >> 4;
 	_uvBlockHeight = (height + 15) >> 4;
 
 	// The planes are sized according to the number of blocks
-	_curPlanes[0] = new byte[_yBlockWidth  * 8 * _yBlockHeight  * 8]; // Y
+	_curPlanes[0] = new byte[_yBlockWidth * 8 * _yBlockHeight * 8];   // Y
 	_curPlanes[1] = new byte[_uvBlockWidth * 8 * _uvBlockHeight * 8]; // U, 1/4 resolution
 	_curPlanes[2] = new byte[_uvBlockWidth * 8 * _uvBlockHeight * 8]; // V, 1/4 resolution
-	_curPlanes[3] = new byte[_yBlockWidth  * 8 * _yBlockHeight  * 8]; // A
-	_oldPlanes[0] = new byte[_yBlockWidth  * 8 * _yBlockHeight  * 8]; // Y
+	_curPlanes[3] = new byte[_yBlockWidth * 8 * _yBlockHeight * 8];   // A
+	_oldPlanes[0] = new byte[_yBlockWidth * 8 * _yBlockHeight * 8];   // Y
 	_oldPlanes[1] = new byte[_uvBlockWidth * 8 * _uvBlockHeight * 8]; // U, 1/4 resolution
 	_oldPlanes[2] = new byte[_uvBlockWidth * 8 * _uvBlockHeight * 8]; // V, 1/4 resolution
-	_oldPlanes[3] = new byte[_yBlockWidth  * 8 * _yBlockHeight  * 8]; // A
+	_oldPlanes[3] = new byte[_yBlockWidth * 8 * _yBlockHeight * 8];   // A
 
 	// Initialize the video with solid green
-	memset(_curPlanes[0],   0, _yBlockWidth  * 8 * _yBlockHeight  * 8);
-	memset(_curPlanes[1],   0, _uvBlockWidth * 8 * _uvBlockHeight * 8);
-	memset(_curPlanes[2],   0, _uvBlockWidth * 8 * _uvBlockHeight * 8);
-	memset(_curPlanes[3], 255, _yBlockWidth  * 8 * _yBlockHeight  * 8);
-	memset(_oldPlanes[0],   0, _yBlockWidth  * 8 * _yBlockHeight  * 8);
-	memset(_oldPlanes[1],   0, _uvBlockWidth * 8 * _uvBlockHeight * 8);
-	memset(_oldPlanes[2],   0, _uvBlockWidth * 8 * _uvBlockHeight * 8);
-	memset(_oldPlanes[3], 255, _yBlockWidth  * 8 * _yBlockHeight  * 8);
+	memset(_curPlanes[0], 0, _yBlockWidth * 8 * _yBlockHeight * 8);
+	memset(_curPlanes[1], 0, _uvBlockWidth * 8 * _uvBlockHeight * 8);
+	memset(_curPlanes[2], 0, _uvBlockWidth * 8 * _uvBlockHeight * 8);
+	memset(_curPlanes[3], 255, _yBlockWidth * 8 * _yBlockHeight * 8);
+	memset(_oldPlanes[0], 0, _yBlockWidth * 8 * _yBlockHeight * 8);
+	memset(_oldPlanes[1], 0, _uvBlockWidth * 8 * _uvBlockHeight * 8);
+	memset(_oldPlanes[2], 0, _uvBlockWidth * 8 * _uvBlockHeight * 8);
+	memset(_oldPlanes[3], 255, _yBlockWidth * 8 * _yBlockHeight * 8);
 
 	initBundles();
 	initHuffman();
 }
 
 BinkDecoder::BinkVideoTrack::~BinkVideoTrack() {
 	for (int i = 0; i < 4; i++) {
-		delete[] _curPlanes[i]; _curPlanes[i] = 0;
-		delete[] _oldPlanes[i]; _oldPlanes[i] = 0;
+		delete[] _curPlanes[i];
```

Comment:
Controversial change. Rules need to be enhanced.

Reply:
Can we rethink about this?
I really think that having two statements in two line is risky, especially since we don't have mandatory braces.

Therefore, we might have something like:
`i++; j++;`
And someone says - hey - that needs to be only on specific conditions, and changes it to:

    if (nice_condition)
        i++; j++;

And here goes nothing...

Solution?
Either enforce having a `{` after every `if`, or disallow two statements in one line.
IMO the second is better:
- The first change is even more controversial than the second change
- Not permitting two statements in one line is a common practice (at least, in my experience, but YMMV)